romancal/datamodels/open_impl.py

```diff
@@ -6,6 +6,7 @@
 import asdf
```

Comment:
Why are we still importing asdf? 
In line 48 I think the asdf.open should be replaced with something like
stdatamodels.DataModel.from_asdf('filename')

Reply:
We're importing asdf so that we can open the file and identify the model class that we should be using with that file.  stdatamodels is a generic package, so it doesn't have any knowledge of romancal or jwst DataModel subclasses.  It wouldn't know which model class to choose, or where to import them from.

Discussion with @perrygreenfield today may lead to a more convenient solution to this...

Reply:
OK, thanks